### PR TITLE
MMB: add "Lounge" whitelabel scene

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: packages/shared/src/WebView.js [hunk 1]
```diff
@@ -21,12 +21,16 @@ export default class WebView extends React.Component<Props> {
...
-  render = () => (
-    <NativeWebView
-      bounces={false}
-      startInLoadingState={false} // it feels faster because we don't have to wait until the page is fully loaded
-      renderError={this.renderError}
-      {...this.props}
-    />
-  );
+  render = () => {
+    console.warn(this.props.source);
```
Comment (mrtnzlml) [inline packages/shared/src/WebView.js]: I added this on purpose. Not sure if it's a good idea though...

File: app/MMB/src/scenes/tripServices/TripServices.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+              ) {
+                booking(id: $bookingId) {
+                  availableWhitelabeledServices {
+                    ...LoungeMenuItem
+                  }
+                }
+              }
+            `}
+            variables={{
+              departureTime: new Date('2019-01-12'), // FIXME
```
Comment (mrtnzlml) [inline app/MMB/src/scenes/tripServices/TripServices.js]: UPS - I need to fix this! 😄

File: app/MMB/src/components/CountryFlag.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { StyleSheet, NetworkImage } from '@kiwicom/mobile-shared';
+
+import type { CountryFlag as CountryFlagType } from './__generated__/CountryFlag.graphql';
+
+type Props = {|
+  data: CountryFlagType,
```
Comment (tbergq) [inline app/MMB/src/components/CountryFlag.js]: could this be ready only type

File: app/MMB/src/scenes/tripServices/LoungeMenuItem.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+  openLoungeLink = (whitelabelURL: string) => {
+    this.setState(
+      {
+        popupVisible: false,
+      },
+      () =>
+        this.props.onOpenWebview(
+          whitelabelURL +
+            (whitelabelURL.indexOf('?') === -1 ? '?' : '&') +
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/LoungeMenuItem.js]: how about using whitelabelURL.includes('?')

File: packages/shared/src/WebView.js (hunk 1 above)
Comment (tbergq) [reply packages/shared/src/WebView.js]: I am not a very big fan of this yellow noisy warnings. How about a console.log instead, less obtrusive?

Comment (mrtnzlml) [reply packages/shared/src/WebView.js]: I will remove it completely.

File: app/MMB/src/scenes/tripServices/TripServices.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+          title={<Translation id="mmb.trip_services.local_services.parking" />}
+          onPress={VoidAction}
+          icon={<TextIcon code="&#xe03e;" />}
+        />
+        <MenuItem
+          title={
+            <Translation id="mmb.trip_services.local_services.transportation" />
+          }
+          onPress={VoidAction}
+          icon={<TextIcon code="<" />} // FIXME: icon?
```
Comment (tbergq) [inline app/MMB/src/scenes/tripServices/TripServices.js]: Should this also be fixed, or is it for another PR?

Comment (mrtnzlml) [reply app/MMB/src/scenes/tripServices/TripServices.js]: No, this thing is that I don't know what icons should I use here. I will remove it as well though and I will consider it to be fixed.

**Comment by mrtnzlml** [comment]
@tbergq Round 2...

**Pull request merged**